src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,97 @@
+expected_output = {
+    "rbp_policies": {
+        1: {
+            "policy_name": "default",
+            "action_policy": "Permit",
+            "action_policy_group": "IP-00"
+        },
+        2: {
+            "src_grp_id": 42,
+            "src_grp_name": "Untrusted",
+            "unknown_group": "Unknown",
+            "action_policy": "Deny",
+            "action_policy_group": "IP-00"
+        },
+        3: {
+            "src_grp_id": 199,
+            "src_grp_name": "FAILED_AUTH",
+            "unknown_group": "Unknown",
+            "action_policy": "Deny",
+            "action_policy_group": "IP-00"
+        },
+        4: {
+            "src_grp_id": 22,
+            "src_grp_name": "Untrusted",
+            "dst_group_id": 22,
+            "dst_group_name": "Untrusted",
+            "action_policy": "Deny",
+            "action_policy_group": "IP-00"
+        },
+        5: {
+            "src_grp_id": 100,
+            "src_grp_name": "Tech",
+            "dst_group_id": 6,
+            "dst_group_name": "Trusted",
+            "policy_groups": {
```

Comment:
no issue to have list at leaf of schema. so, can change to below to be simpler?
```
"policy_groups": ['NFS-08', 'ACCESS-01']

Reply:
Updated to use lists.